src/environments/open_spiel.jl

```diff
@@ -1,26 +1,18 @@
-module OpenSpielWrapper
```

Comment:
I must admit I'm a lazy man... Do you use any tool to generate this line?

Reply:
Haha, no, I felt very stupid. The approach with the module is actually really nice. I just didn't find a way quickly to get Requires to work with a module. I hope we can move back to a more elegant solution, once https://github.com/JuliaLang/Pkg.jl/issues/1285 gets resolved.